### DISCUSSION TITLE
How to install AdGuardHome in openwrt

### DISCUSSION THREAD

**Discussion by ZeroBlock0** [AdguardTeam/AdGuardHome#6144] [Q&A]
How to install AdGuardHome in openwrt

**Comment by YBS-PC** [comment]
The same question. I have openwrt on rockchip aarch64 and if I understand correctly, then I should use AdGuardHome_linux_arm64.tar.gz . But the file that is installed from the openwrt repository is different (not same hash) from the same Adguard home file from this archive with the same version.

**Comment by ghost** [comment]
The arm64 version should work from the AGH repo, but the automated installer should also work.
